icxcli/icx/utils.py

```diff
@@ -305,13 +264,25 @@ def check_balance_enough(balance, amount, fee):
 
 
 def check_amount_and_fee_is_valid(amount, fee):
```

Comment:
No unit test for this function. Please implement this one for all cases you tested.

Reply:
Change function name as ```def check_amount_and_fee_are_valid()```